subprojects/internal-performance-testing/src/integTest/groovy/org/gradle/performance/fixture/MavenDownloaderTest.groovy

```diff
@@ -37,6 +38,9 @@ class MavenDownloaderTest extends Specification {
     def setup() {
         installRoot = tmpDir.newFolder()
         downloader = new MavenInstallationDownloader(installRoot)
+        if (JavaVersion.current().isJava7()) {
```

Comment:
Is this still required?

Reply:
Yes, that test actually uses a standard JDK connection for the download.